pkgs/by-name/lw/lwk/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lwk";
+  version = "0.2.0";
+  cargoBuildFlags = "--bin lwk_cli";
+
+  src = fetchFromGitHub {
+    owner = "Blockstream";
+    repo = pname;
+    rev = "f1e1473b20f0699c9419986094e094786cb30be4";
```

Comment:
Please use the equivalent tag (`"refs/tags/cli_${version}"` in this case, I suppose) when using a release version